jraft-core/src/main/java/com/alipay/sofa/jraft/util/RepeatedTimer.java

```diff
@@ -112,7 +112,6 @@ public void runOnceNow() {
             if (this.timerTask != null && this.timerTask.cancel()) {
                 this.timerTask = null;
                 run();
-                schedule();
```

Comment:
这个地方保留与否虽然对执行结果没有影响，但是其实从语义上来说是不同的逻辑。
当删除掉这个 ``schedule()`` 方法之后，schedule() 调用链的源头就在 run() 中。
如果保留这个 ``schedule()`` 方法，那么 run() 中的 task 会被 cancel，调用链的源头就在 ``runOnceNow()``，正如注释所言 "it will cancel the timer and re-schedule it."
所以我更倾向于保留这个 schedule，这也符合这个方法的本意。

Reply:
这个方法没用到，倒也不用纠结，甚至我觉的可以删除

Reply:
@killme2008 我也同感，所以我也是这么与 @fengjiachun